javascript/javascript1/week1/week1.js

```diff
@@ -0,0 +1,68 @@
+//Age-ify (A future age calculator)
+console.log("********** Age-ify (A future age calculator) *************");
+const yearOfBirth = 1991;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+console.log("******************* Goodboy-Oldboy (A dog age calculator) **********");
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+let dogYear;
+const shouldShowResultInDogYears = false;
+if(shouldShowResultInDogYears){
+    dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+    console.log("Your dog will be "+ dogYear + " dog years old in " +dogYearFuture);
+}
+else{
+    dogYear = (dogYearFuture - dogYearOfBirth);
+    console.log("Your dog will be "+ dogYear + " human years old in " +dogYearFuture);
```

Comment:
Experiment moving the `console.log` outside of the `if else` statement. Maybe you can have just one `console.log` and avoid a the repetition.